dock_header.go

```diff
@@ -49,6 +50,11 @@ type dockHeader struct {
 	dragInsertIndex       int
 }
 
+func (d *dockHeader) CurrentDockTabLabel() *Label {
```

Comment:
Why is this needed? If you want to know the title of the current tab, you can call `.CurrentDockable().Title()`. There is no need to access it from here.